assembl/static2/js/app/graphql/mutations/createPost.graphql

```diff
@@ -1,15 +1,16 @@
-mutation createPost($ideaId: ID!, $subject: String, $body: String!) {
-  createPost(ideaId:$ideaId, subject: $subject, body: $body) {
+mutation createPost($ideaId: ID!, $subject: String, $body: String!, $parentId: ID) {
+  createPost(ideaId:$ideaId, subject: $subject, body: $body, parentId: $parentId) {
     post {
-      ... on PropositionPost {
+      ... on AssemblPost {
```

Comment:
If you want to reuse this createPost mutation, you should actually keep the PropositionPost fragment and write the same for Post fragment (you should remove the AssemblPost objecttype in the backend code)